common/templatetags/snippets.py

```diff
@@ -13,7 +13,7 @@ def header(context):
 @register.inclusion_tag('common/footer.html', takes_context=True)
 def footer(context):
     return {
-        'footer': Footer.objects.all()[0],
+        'footer': Footer.objects.get(id=1),  # Titled 'COS Footer', the home page footer.
```

Comment:
I would prefer `Footer.objects.get(title='COS Footer')` rather than just the first footer in the db, seems like that's a better unique identifier, in case Footers were removed from the db and added again.  But this seems unlikely so this is fine.

Reply:
lol this is nice. Would there be any issue with having mutiple sites? i.e would the non-primary site not get the wrong footer here? Not that it was with [0]. We might need to filter this somehow.

Don't remember - this was a long time ago, just raising the point.

Reply:
This certainly isn't ideal, but it's better than the status quo (wrong footer). I suppose it'd be a bit more safe to do

```python
Footer.objects.order_by('pk').first()
```

in case we ever delete the Footer with id=1.

That said, I think this is an OK workaround for now.